lib/logmsg/tags.c

```diff
@@ -170,6 +170,7 @@ log_tags_reinit_stats(void)
 {
```

Comment:
Optional:
`log_tags_reinit_stats` can be static function? If I saw correctly this function is only called in this file. It's not important just to make sure this function can be called by application hook.

Reply:
It could be static. Currently, it is only used by an app hook.